### PR TITLE
Detective's Revolver Reskin+Cleaning up the header

### PR DESCRIPTION
**Pull request by Chronitonity**
Adds Markoboy's new Revolver sprite, the Blue Silver, based on the best revolver in the world, the R8.

http://puu.sh/lRMDp/39ff176e8f.png

That part of the PR is done entirely for him, hence his name being in the changelog.

Secondly the changelog header has been cleared up a bit, moving names around, and removing ones we don't need.

It also gets rid of the "recently rebased" message because that happened forever ago.

I have no idea who the current mentors are so a formatted list delivered to me would be great for the next time i clean this header up.

### CODE REVIEW

**Comment by danhibiki** [comment]
+1

**Comment by ghost** [comment]
tbh r8 is bad
though yes this is just a reskin so yeah put it in plz for meme

File: code/modules/projectiles/guns/projectile/revolver.dm [hunk 1]
```diff
@@ -110,6 +110,7 @@
 	options["Black Panther"] = "detective_panther"
 	options["Gold Trim"] = "detective_gold"
 	options["The Peacemaker"] = "detective_peacemaker"
+	options["Blue Sliver"] = "detective_bluesilver"
```
Comment (SplergeBucket) [inline code/modules/projectiles/guns/projectile/revolver.dm:113]: Blue Sliver =/= detective_bluesilver

**Comment by Lawligagger** [comment]
Sure

**Pull request merged**